src/frontend/src/components/AddUser/index.tsx

```diff
@@ -93,17 +99,12 @@ class AddUser extends React.Component<Props, State> {
                 name="name"
                 type="text"
                 className="w-100 pa3 br2 b--black-20"
-                required={true}
```

Comment:
I still don't know this: what does the `required` property do and what does removing it do? It only has to do with the aria, right?

Reply:
It's for screen readers to announce that it's required. Please see https://developer.paciellogroup.com/blog/2019/02/required-attribute-requirements

I think `required` should be static and paired with `aria-invalid ` and `aria-describedby`, but I don't have experience with validation.

Reply:
I see. I removed the required property and ran the app and did not see any difference in how the validation functioned. I'll put them all back. Thanks for pointing that out.

Reply:
I just checked the User component and it does already have a button to take you to the edit-user page just like the new user button in the Userlist component. If it is not working I'll have to take another look into it.

Reply:
Ah! I see clicking the name works, but I was only trying the pencil-icon. I think that button is what we should have for editing that user's info, and not make the person's name a link.

@hmarcks I checked your mockup and saw you didn't put the edit icon next to users, only groups. @ForrestLonganecker and I decided some time ago that we liked the idea of using the pencil icon for users also. What do you think?

Reply:
I thought the link was a bit more appropriate since you may just be looking at their profile for more info and not necessarily editing, especially if we expose the user list to non-admins in the future, but I'm fine with either.

Reply:
@hmarcks Ah that make sense. Do you have something in mind for what a non-editor's view of a person's profile would look like? I don't think that's in the prototype. 

Basically the only information that isn't included in the table on the admin page is their email address -- maybe it just belongs there? This would require that the user list stay hidden from non-admins.

Reply:
I don't have that view and we may never want it. I think it could be fine to show the email in the admin list and use the edit button. It could get to be a lot of info if there are a lot of users viewed on a small screen, but we can always change the layout